### ISSUE TITLE
1.8.8 Cover Art Not Loading

### ISSUE THREAD

**Issue by 0pLuS0** [DeaDBeeF-Player/deadbeef#2772]
### Steps to reproduce the problem
I edited the column and changed one section to Album Art, then went to, File > Add Folders when they were done loading, the Album Art section was blank, I cleared it several times and I tried adding the folders several times, but the covers never appeared.

### What's going on? Describe the problem in as much detail as possible.
Album Art doesn't appear, the section is blank. I assumed, all I had to do was add the Album Art column and load folders and it worked out the box, but it's not.

I have this working in 0.7.2, but it was compiled for the GTK2 plugin, and if I start 1.8.8 with the same playlists and config file from 0.7.2, it will then display the Album Art. 

I read somewhere, you mentioning in the past bad image formats, my jpg images are fine, as I just mentioned the Album Art works in 0.7.2, and 1.8.8 will work, as long as I use the 0.7.2 playlist and config. I did not embed the images into the mp3, I have them in the directory with the mp3 files named as; cover.jpg

Here's a screen shot showing, as an example how all my music directories look, with the mp3 files, and a cover.jpg image.

![cover jpg](https://user-images.githubusercontent.com/36190223/160747263-ccf0d5f4-ad6d-4db9-818e-fcf37cf62ec1.jpg)
### Information about the software:

Deadbeef version: 1.8.8
OS: Slackware 
GTK: gtk+3-3.24.31

I compiled 1.8.8 from source using a build script, with these compile options, and here is the Plugin Summary.

export CC=${CC:-clang}
export CXX=${CXX:-clang++}
SLKCFLAGS+=" -msse2"

CFLAGS="$SLKCFLAGS" \
CXXFLAGS="$SLKCFLAGS" \
./configure \
  --prefix=/usr \
  --libdir=/usr/lib${LIBDIRSUFFIX} \
  --docdir=/usr/doc/$PRGNAM-$VERSION \
  --disable-gtk2 \
  --disable-nls \
  --enable-shared \
  --build=$ARCH-slackware-linux

make V=1
make install DESTDIR=$PKG

## Plugin Summary:

    aac: no - AAC player (m4a, aac, mp4) based on FAAD2
    adplug: yes - adplug player (OPL2/OPL3 emulation)
    alac: yes - ALAC plugin
    alsa: yes - ALSA output
    artwork: yes - Cover art plugin
    cdda: yes - cd audio player
    converter: yes - plugin for converting files to any formats
    coreaudio: no - CoreAudio output plugin
    dca: yes - libdca (DTS Audio) player plugin
    dsp_src: yes - High quality samplerate conversion using libsamplerate
    dumb: yes - DUMB module plugin, for MOD, S3M, etc
    ffap: yes - Monkey's audio (APE) decoder
    ffmpeg: yes - ffmpeg codecs
    flac: yes - flac player
    gme: yes - chiptune music player based on GME
    gtkui3: yes - GTK3 user interface
    gtkui: no - GTK2 user interface
    hotkeys: yes - Local and global hotkeys support
    lastfm: yes - last.fm scrobbler
    m3u: yes - M3U and PLS playlist support
    mms: yes - mms streaming support
    mono2stereo: yes - mono2stereo DSP plugin
    mp3: yes - mp3 plugin
    libmad: yes - libmad backend for mp3 plugin
    libmpg123: yes - libmpg123 backend for mp3 plugin
    musepack: yes - musepack player plugin
    notify: yes - notification-daemon support plugin
    nullout: yes - NULL output
    opus: yes - ogg opus player
    oss: yes - oss output plugin
    pltbrowser: yes - playlist browser gui plugin
    psf: yes - PSF player, using Audio Overload SDK
    pulse: yes - PulseAudio output plugin
    rgscanner: yes - plugin for ReplayGain scanner support
    sc68: yes - sc68 Atari ST And Amiga player
    shellexec: yes - shell commands plugin
    shellexecui: yes - GTK user interface for setting up shellexec plugin
    shn: yes - SHN plugin based on xmms-shn
    sid: yes - SID player based on libsidplay2
    sndfile: yes - PCM (wav,aiff,etc) player based on libsndfile
    soundtouch: yes - SoundTouch plugin used for time stretching and pitch shifting
    stdio: yes - Standard IO plugin
    supereq: yes - Equalizer based on Super EQ library by Naoki Shibata
    tta: yes - TTA player plugin
    vfs_curl: yes - http/ftp streaming support
    vfs_zip: yes - zip archive support
    vorbis: yes - ogg vorbis player
    vtx: yes - vtx file player (ay8910/12 emulation)
    wavpack: yes - wavpack player
    wildmidi: yes - WildMidi player plugin
    wma: yes - WMA plugin

Here's a screen shot of 1.8.8, in this screen shot I used Design Mode and cleared it and then added in a playlist with no tabs, before doing this, I just tried adding Album Art with no changes to deadbeef, and it still didn't work.

THANKS

![deadbeef-1 8 8](https://user-images.githubusercontent.com/36190223/160746119-17871e42-780b-4d24-922e-5638715a5255.jpg)

**Comment by Alexey-Yakovenko** [comment]
Right click on any playlist column header, and enable grouping.

**Comment by 0pLuS0** [comment]
@Alexey-Yakovenko 

Ahhhh I'm really sorry if this is mentioned somewhere, I certainly forgot about it.

Something I noticed, with the Track Number, the numbers didn't appear all the way over to the left starting under the Letter T for Track, they were more positioned over to the right, and when moving the Title section, the numbers moved with it.

So I removed the Track Number and added it again, now the numbers position themself correctly, not sure if you are aware of this?

THANKS
